packages/wallets/src/features/cashier/modules/WithdrawalCrypto/components/WithdrawalCryptoReceipt/WithdrawalCryptoReceipt.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import { WalletButton, WalletCard, WalletText } from '../../../../../../components';
+import ArrowDown from '../../../../../../public/images/ic-back-arrow.svg';
+import { WithdrawalCryptoDestinationAddress } from './components';
+import './WithdrawalCryptoReceipt.scss';
+
+type TProps = {
+    onClose: () => void;
+    withdrawalReceipt: {
+        address: string;
+        amount: string;
+        currency?: string;
+    };
+};
+
+const WithdrawalCryptoReceipt: React.FC<TProps> = ({ onClose, withdrawalReceipt }) => {
+    const history = useHistory();
+    const { address, amount, currency } = withdrawalReceipt;
+
+    return (
+        <div className='wallets-withdrawal-crypto-receipt'>
+            <div className='wallets-withdrawal-crypto-receipt__accounts-info'>
+                <WalletCard
+                    balance={`-${amount} ${currency}`}
+                    currency={currency ?? ''}
+                    iconSize='md'
+                    landingCompanyName='SVG'
+                />
+                <ArrowDown />
+                <WithdrawalCryptoDestinationAddress address={address} />
+            </div>
+            <div className='wallets-withdrawal-crypto-receipt__withdrawal-info'>
+                <WalletText align='center' size='xl' weight='bold'>
+                    {amount} {currency}
+                </WalletText>
+                <WalletText align='center' as='p'>
+                    Your withdrawal is currently in review. It will be processed within 24 hours. We&rsquo;ll send you
+                    an email once your transaction has been processed.
+                </WalletText>
+            </div>
+            <div className='wallets-withdrawal-crypto-receipt__actions'>
+                <WalletButton
+                    color='white'
+                    onClick={() => history.push('/wallets/cashier/transactions')}
```

Comment:
@aum-deriv 
we are removing the code using in onClose func. 
should we also run onClose in current onclick when we are moving to transactions?

Reply:
Hi @heorhi-deriv, since we are unmounting the entire `WalletWithdrawal` component when we open transactions, the state `verificationCode` goes out-of-scope.
That's why when we return to the withdrawal-tab, we see the email-verification page.